automatic/logfusion/logfusion.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>logfusion</id>
+    <version>0.0</version>
+    <title>LogFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>http://www.logfusion.ca</projectUrl>
+    <licenseUrl>https://www.logfusion.ca/License/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/7457c4aec7a50384fc3e8787fb59e4be48b84738/icons/LogFusion.png</iconUrl>
+    <docsUrl>https://www.logfusion.ca/Help/</docsUrl>
+    <mailingListUrl>https://www.logfusion.ca/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ## Features
+    - Pwerful log monitoring
+    - Row highlighting and text filtering
+    - Custom columns to parse and split your logs
+    - Auto-scroll and always show you the newest lines
+    - Load any text logs, including web server logs, developer debug logs and any other text that is outputted to a file
+    - Read Event Logs in real-time
+    </description>
+    <copyright>Copyright © 2017 Binary Fortress Software</copyright>
+    <tags>logfusion log logs parsing tail monitoring trial freeware</tags>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatci/logfusion</packageSourceUrl>
```

Comment:
You have a typo here.
change it to `https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/logfusion`